addons/mute-project/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Muted project player mode",
-  "description": "Ctrl+Click the green flag to mute/unmute the project.",
+  "name": "Project player volume control",
+  "description": "Ctrl+Click the green flag to mute/unmute the project, or use the slider",
```

Comment:
The description should end with a period. Also, just calling it "slider" is unclear.